Gemfile.lock

```diff
@@ -122,11 +131,21 @@ GEM
     diff-lcs (1.3)
     erubis (2.7.0)
     execjs (2.7.0)
+    factory_girl (4.2.0)
+      activesupport (>= 3.0.0)
+    factory_girl_rails (4.2.0)
+      factory_girl (~> 4.2.0)
+      railties (>= 3.0.0)
```

Comment:
Hmmm this might be an issue. During my development, I was using factory_girl_rails gem as well, and there's a bit of a version difference. My code:

`factory_girl (4.8.0)`
`activesupport (>= 3.0.0)`
`factory_girl_rails (4.8.0)`
`factory_girl (~> 4.8.0)`
`railties (>= 3.0.0)`